goose.go

```diff
@@ -1200,6 +1228,15 @@ func (ctx Ctx) expr(e ast.Expr) coq.Expr {
 	return ctx.exprSpecial(e, false)
 }
 
+func (ctx Ctx) funcLit(e *ast.FuncLit) coq.FuncLit {
+	fl := coq.FuncLit{} // TODO: Add type checking?
```

Comment:
You don't need to add any typechecking, that code is mostly dead now (the typechecker and type system don't actually scale to all of our examples any more).